telemetry/definitions/commonDefinitions.json

```diff
@@ -2024,6 +2027,15 @@
                 { "type": "lambdaArchitecture", "required": false }
             ]
         },
+        {
+            "name": "sam_info",
+            "description": "Called when checking if the SAM executable on the local machine is valid with a valid version",
+            "metadata": [
+                { "type": "result" },
+                { "type": "reason", "required": false },
+                { "type": "version", "required": false }
+            ]
```

Comment:
this is already on mainline? https://github.com/aws/aws-toolkit-common/blob/d955c40b22b79f8b6b4b3ba8bdb88e0b1ba99f1c/telemetry/definitions/commonDefinitions.json#L2031-L2037

Reply:
Yes, they weren't being reflected as merged on mainline which is why showing up in the diff here.